index.js

```diff
@@ -1,33 +1,32 @@
 
-
-//Getting the different placeholders
-
 let content = document.getElementById("content");
 let integerValue = parseInt(content.innerHTML);
 
-const decreaseButton = document.getElementById("btn-decrease");
-const resetButton = document.getElementById("btn-reset");
-const increaseButton = document.getElementById("btn-increase");
-
-
-//Increasing value when the increase button is clicked on
-increaseButton.addEventListener("click", () => { 
-    integerValue += 1;
-    content.innerHTML = integerValue;
+const arrayButton = document.getElementsByTagName("button");
+const buttons = Object.entries(arrayButton);
+buttons.forEach((button, index) => {
+    if (index === 0) { //Decrease by 1 for index 0
```

Comment:
It's better to write a separate function that takes the index of the buttons' array as argument and then inside the function you could use switch statement to handle the operation(increase, decrease, reset).